src/MainWindow.vala

```diff
@@ -165,10 +184,12 @@ public class Installer.MainWindow : Gtk.Dialog {
             stack.visible_child = try_install_view;
         });
 
-        disk_view.next_step.connect (() => load_encrypt_view ());
+        disk_view.next_step.connect (() =>  load_encrypt_view ());
+
+        load_check_view ();
     }
 
-    private void load_partitioning_view () {
+    private void load_partitioning_view() {
```

Comment:
space before parens :smile:

Reply:
I'll be using `vala-lint` to fix all the style issues